README.md

```diff
@@ -43,18 +43,19 @@ $ python server.py
 ```
 ## API
 
-There are six endpoints:
+There are seven endpoints:
  * `GET /invalid-responses` - returns a json response of all invalid survey responses in the connected database
  * `POST /queue` - Publishes a message to a corresponding rabbit message queue based on the message content. Returns a 200 response and JSON value `{"result": "ok"}` if the publish succeeds or a 500 response with JSON value `{"status": 500, "message": <error>}` if it does not.
  * `GET /healthcheck` - returns a json response with key/value pairs describing the service state
  * `POST /responses` - store a json survey response
  * `GET /responses` - retrieve a JSON response of all valid survey responses in the connected responses.
  * `GET /responses/<tx_id>` - retrieve a survey by id
  * `DELETE /responses/old` - delete responses older than a number of days set in config 
+ * `GET /feedback/<feedback_ID>` - retrieve a JSON response of a valid ID
```

Comment:
I'm not sure why this requires a new end point?

I believe feedback submissions are treated the same as normal submissions, so we should be able to use the responses endpoint.

Reply:
I'm slightly wrong, feedback submissions are stored in a separate table and keyed by an integer id rather than the tx_id.

However it we shouldn't have one endpoint for adding data and two endpoints for retrieving data. It's actually a bit odd thats sending requests to the /responses endpoint to save a feedback submission

Therefore I suggest adding  an additional end point

`POST /feedback` - store a json feedback response.

We should also change the database table and store tx_id similar to the response table. Then these endpoints should use the tx_id to store and retrieve a feedback response.

If we return the tx_id then we can probably drop the use of id_tag everywhere else

Reply:
As discussed please add a ticket to the backlog to add the missing end point for storing feedback